packages/sources/pound-token/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@chainlink/pound-token-adapter",
```

Comment:
Probably worth a discussion but my opinion is that we name this after the API data source, which would be `bank-frick-adapter`